lib/src/animated_sampler.dart

```diff
@@ -174,7 +174,7 @@ class _RenderShaderSamplerBuilderWidget extends RenderProxyBox {
 
   @override
   void paint(PaintingContext context, Offset offset) {
-    if (size.isEmpty) {
+    if (size.isEmpty || !_enabled) {
```

Comment:
I don't think this is quite right - because now the child doesn't render if its not enabled-  instead of the child rendering normally without the shader. Instead I would change the line below:

```
assert(!_enabled || offset == Offset.zero);
```

Reply:
I made this change, but be aware it's not tested.

Reply:
Did some basic testing. Seems to work (at least with my scenario).